keras_cv/layers/object_detection_3d/voxelization.py

```diff
@@ -136,28 +128,28 @@ def call(
 
         # [B, N, dim]
         # convert point voxel to positive voxel index
-        point_voxel_xyz = (
-            point_voxel_xyz_int - voxel_origin[tf.newaxis, tf.newaxis, :]
+        point_voxel_xyz = point_voxel_xyz_int - ops.cast(
+            ops.expand_dims(ops.expand_dims(voxel_origin, axis=0), axis=0),
```

Comment:
nit, I think `expand_dims(x, axis=(0, 1))` should work and maybe be a bit more readable.

Reply:
Done